ursa/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Determine which set of files to use depending on the type of run
+if [ "$1" != "" ] && [ "$1" != "tuned" ] && [ "$1" != "tuned-memory" ]; then
+    echo "Error: command line argument must be one of {'', 'tuned', 'tuned-memory'}"
+    exit 1
+else if [ ! -z "$1" ]; then
+    SUFFIX="-$1"
+fi
+fi
+
+TRIES=3
+QUERY_NUM=1
+cat queries"$SUFFIX".sql | while read query; do
```

Comment:
This is executing Q28 incorrectly (see #252)
```suggestion
cat queries"$SUFFIX".sql | while read -r query; do
```